src/main/java/com/onevizion/scmdb/dao/DbScriptDaoOra.java

```diff
@@ -50,14 +51,19 @@ public class DbScriptDaoOra extends AbstractDaoOra {
         return dbScripts;
     };
 
-    public Map<String, SqlScript> readMap() {
+    @Cacheable("dbScripts")
+    public Map<String, SqlScript> loadDbScriptsMapCached() {
         return jdbcTemplate.query(READ_ALL, dbScriptsExtractor);
     }
 
     public Long readCount() {
         return jdbcTemplate.queryForObject(READ_COUNT, Long.class);
     }
 
+    public String readScriptTextById(Long id) {
+        return (String) jdbcTemplate.queryForObject(READ_SCRIPT_TEXT, String.class, id);
```

Comment:
`(String)` is meaningless here, remove it

Reply:
Done. Removed the entire method because it became unused.